ui/app/views/demo.html

```diff
@@ -64,7 +64,7 @@ <h2 class="col-md-12">{{model.demo.name}} <a class="btn btn-primary btn-xs" ng-h
         <autocomplete ng-model="model.additionalBug.assignee" data="model.users" click-activation="true" ></autocomplete>
       </fieldset>
       <div class="col-md-12">
-        <a class="col-md-1" href ng-click="resetBugForm(); showBugForm = false">Cancel</a>
+        <a class="btn btn-default btn-xs" href ng-click="resetBugForm(); showBugForm = false">Cancel</a>
```

Comment:
Funny, the Cancel is an 'a', but Add a 'button'. Because of default behavior maybe?